### DISCUSSION TITLE
FreeType is now 2.12.1,Whether the document should be updated?

### DISCUSSION THREAD

**Discussion by xland** [mikke89/RmlUi#378] [General]
FreeType is now 2.12.1
Whether the document should be updated?
https://mikke89.github.io/RmlUiDoc/pages/cpp_manual/building_with_cmake.html
I've tested it manuly.It worked well.

**Comment by mikke89** [comment]
Thanks for letting me know. I'm currently working on some larger updates to the docs for RmlUi 5.0, I'll make sure to update this.

**Reply by xland** [reply]
Will RmlUi 5.0 be released soon?

**Reply by xland** [reply]
I'm willing to be one of the first users of this version

**Reply by mikke89** [reply]
Yep, shouldn't be too long until I release it now.
